.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: pass-authz Continuous Integration
+on: [pull_request]
```

Comment:
With just a single condition, you could drop the `[]` if you wanted.

Reply:
will do.... copy and paste bug :-)